trie/transition.go

```diff
@@ -20,18 +20,21 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/ethdb"
+	"github.com/ethereum/go-ethereum/rlp"
 	"github.com/gballet/go-verkle"
 )
 
 type TransitionTrie struct {
 	overlay *VerkleTrie
 	base    *SecureTrie
+	storage bool
 }
 
-func NewTransitionTree(base *SecureTrie, overlay *VerkleTrie) *TransitionTrie {
+func NewTransitionTree(base *SecureTrie, overlay *VerkleTrie, st bool, sig string) *TransitionTrie {
```

Comment:
Does the fact that `sig` is an unused parameter something to be considered? (Or maybe planning already something else?)

Reply:
it should have been removed, I will push a fix to this effect in a few minutes